src/components/containers/seo.tsx

```diff
@@ -137,15 +156,29 @@ const SEO = ({
                 },
                 {
                     property: 'og:title',
-                    content: meta_attributes?.og_title || default_og_title,
+                    content: meta_attributes?.og_title
+                        ? typeof meta_attributes.og_title === 'string'
+                            ? localize(meta_attributes.og_title)
+                            : localize(
+                                  meta_attributes.og_title.key,
+                                  meta_attributes.og_title.values,
+                              )
+                        : default_og_title,
```

Comment:
Please refactor this and the similar things
```suggestion
                    content: (typeof meta_attributes.og_title === 'string'
                            ? localize(meta_attributes.og_title)
                            : localize(
                                  meta_attributes.og_title.key,
                                  meta_attributes.og_title.values,
                              ))
                        || default_og_title,
```